src/mcts/search.cc

```diff
@@ -1602,6 +2426,62 @@ void SearchWorker::FetchMinibatchResults() {
   }
 }
 
+template <typename Computation>
```

Comment:
Why template if it's only called with one type?

Reply:
Immediate follow up PRs intend to reduce code duplication - one of which will mean this will start being called from the non-multigather path - with a different type. (It was called with two types in 1483 where I copied the function from, I was just keeping this PR's influence into the existing path somewhat minimal for 'clarity of safety'.  The follow up PRs will be focused on the single duplications, making them easy to review without the noise of everything else this PR is adding.